api/app/hfi/prep_template.py

```diff
@@ -0,0 +1,152 @@
+"""Prep cycle pdf template"""
+PREP_TEMPLATE_HTML = """
+'<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Prep Cycle Template</title>
+</head>
+
+<style>
+    td {
+        border-top: 2px solid black;
+        border-bottom: black 2px solid;
+        padding: 0;
+        border-collapse: collapse;
+        min-width: 35px;
+        max-width: 200px;
+        text-align: center;
+    }
+    th{
+        min-width: 35px;
+        max-width: 200px;
+        text-align: center; 
+    }
+
+    table {
+            border-collapse:collapse;
+            border:2px solid #000000;
+        }
+
+    .leftBorder{
+        border-left: 1px solid black;
+        border-collapse: collapse;
+    }
+
+    .rightBorder{
+        border-right: 1px solid black;
+        border-collapse: collapse;
+    }
+
+    .boldRightBorder{
+        border-right: 2px solid black;
+        border-collapse: collapse;
+    }
+
+    .boldLeftBorder{
+        border-left: 2px solid black;
+        border-collapse: collapse;
+    }
+
+    .planningArea {
+        background-color: cornflowerblue;
+        border-bottom: black 2px solid;
+        border-collapse: collapse;
+        height: 20px;
+        padding: 0;
+    }
+    .station{
+        border-bottom: black 2px solid;
+        border-collapse: collapse;
+        padding: 0;
+    }
+</style>
+
+<body>
+    <table>
+        <tr class="leftBorder">
```

Comment:
I found out that inline class styles like this don't work on our ubuntu containers in our cluster :(
Styles can be added to `api/app/hfi/style.css` and then that path passed in to the pdfkit render function, see: https://github.com/bcgov/wps/blob/985e1d5cf7336c9fba5e1c5dcf8c4fcc9d9c1c96/api/app/hfi/daily_pdf_gen.py#L44